.github/CODEOWNERS

```diff
@@ -12,9 +12,9 @@
 /packages/analytics               @literat @crishpeen @curdaj @pavelklibani @lmc-eu/spirit-design-system
 /packages/codemods                @literat @adamkudrna @crishpeen @pavelklibani @curdaj @lmc-eu/spirit-design-system
```

Comment:
who not codemods too? And Katka can be here too imho.

Reply:
Added.